sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -961,7 +1014,9 @@ int main() {
 
   auto exec = g.finalize(q.get_context());
 
-  q.submit(exec).wait();
+  q.submit([&](handler &h) {
+    h.exec_graph(exec);
+  }).wait();
```

Comment:
We might want to use both variants, graph execution through handler and shortcut, for the examples:

```suggestion
  q.exec_graph(exec).wait();
```